src/main/java/mate/academy/spring/controller/IndexController.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.controller;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class IndexController {
+    @GetMapping("/")
+    public String hello(Authentication authentication) {
+        return String.format("Hello, %s!", authentication.getName());
+    }
+
+    @GetMapping("/admin")
+    public String helloAdmin() {
+        return "Hello admin!";
+    }
+
+    @GetMapping("/user")
```

Comment:
forgot to delete?)
or add this mappings in security config

Reply:
first I forgot :) but than I think it is not bad functional, so I moved it to security config

Reply:
First I forgot but then I thought that it a good functional :) but I don't get how to move it to security controller and why to do so? it is a controller, so I think that it has to be in controller package?